docs/topic_guide/channel_response.rst

```diff
@@ -0,0 +1,146 @@
+.. _sunkit-instruments-topic-guide-channel-response:
+
+************************************
+A Vocabulary for Instrument Response
+************************************
+
+This topic guide provides a vocabulary for response functions for narrow-band imaging instruments.
+The goal of providing a single vocabulary for instrument response calculations is to provide a specification for a common interface that can be used by multiple instruments.
+This reduces the amount of effort needed to develop analysis software for new instruments and enables cross-instrument comparisons as upstream packages and users can program against a single interface for these response calculations.
+An abstract implementation of this vocabulary is provided in this package.
+
+Temperature Response
+--------------------
+
+The temperature response describes the instrument sensitivity as a function of temperature.
+It is a useful quantity when performing thermal analysis of imaging data, such as a differential emission measure or filter ratio analysis.
+The temperature response is defined as,
+
+.. math::
+
+    K(T) = \int\mathrm{d}\lambda\,R(\lambda)S(\lambda,T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1} \,\mathrm{cm}^5]
+
+It has a physical type of DN per pixel per unit time per unit emission measure.
```

Comment:
Fair point, especially since the astropy unit definition is not overly descriptive: https://docs.astropy.org/en/stable/units/index.html#module-astropy.units.astrophys.